fs-sim/README.md

```diff
@@ -0,0 +1 @@
+# fs-sim
```

Comment:
We should add some content here.

Reply:
Idem https://github.com/input-output-hk/ouroboros-network/pull/4425#discussion_r1132383639